src/main/java/com/joeyfrazee/nifi/reporting/PluginEnvironmentVariable.java

```diff
@@ -52,6 +52,16 @@ public enum PluginEnvironmentVariable {
     ELASTICSEARCH_USERNAME("ELASTICSEARCH_USERNAME"),
     /** The environment variable defining the password for Elasticsearch authentication. */
     ELASTICSEARCH_PASSWORD("ELASTICSEARCH_PASSWORD"),
+    /**
+     * The environment variable defining the comma-separated list of attributes to include in the
+     * data sent to Elasticsearch.
+     */
+    ELASTICSEARCH_INCLUSION_LIST("ELASTICSEARCH_INCLUSION_LIST"),
+    /**
+     * The environment variable defining the comma-separated list of attributes to exclude from the
+     * data sent to Elasticsearch.
+     */
+    ELASTICSEARCH_EXCLUSION_LIST("ELASTICSEARCH_EXCLUSION_LIST"),
```

Comment:
Why do we need to define both the enum name and a string?

From what I understand the builtin `.name()` method would return the enum name verbatim which is what we want. So the constructor can just be:

```
    PluginEnvironmentVariable() {
        this.name = ENV_VAR_PREFIX + this.name();
        final String rawValue = System.getenv(this.name);
        this.value = Optional.ofNullable(rawValue);
    }
```

Was there a reason we couldn't do that? If not, let's just clean it up to get rid of the duplication. If there was a reason, then let's add a comment explaining it.